ee/tabby-webserver/src/service/auth.rs

```diff
@@ -594,15 +596,23 @@ mod tests {
         }
     }
 
+    async fn test_authentication_service_with_mail() -> (AuthenticationServiceImpl, TestEmailServer)
```

Comment:
@wsxiaoys Needed to add this to fix tests, just wanted to make sure you see this before it's merged